conductor/bridge.py

```
@@ -1557,6 +1558,117 @@ async def slack_cmd_help(ack, respond, command):
+            log.info(
+                "kanban_watch_loop: process exited (code=%s), reconnecting in %ds",
+                proc.returncode, backoff,
+            )
+
+        except asyncio.CancelledError:
+            log.info("kanban_watch_loop: cancelled")
+            if proc is not None and proc.returncode is None:
+                try:
+                    proc.terminate()
+                    await asyncio.wait_for(proc.wait(), timeout=3)
+                except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)